src/bindings/python/tests/test_ngraph/test_create_op.py

```diff
@@ -1962,3 +1962,14 @@ def test_nv12_to_rgb():
     assert node_separate_planes.get_output_size() == 1
     assert node_separate_planes.get_output_element_type(0) == Type.f32
     assert list(node_separate_planes.get_output_shape(0)) == expected_output_shape
+
+
+def test_softsign():
+    input_shape = [2, 4, 8, 16]
+
+    param = ov.parameter(input_shape, name="input")
```

Comment:
I'm a little bit surprised, that it is possible to create `parameter` without specifying `element_type`. 
What is set by default?

Reply:
f32 (or if you want to keep it pythonic np.float32) - this is just a part of friendly PyAPI:
https://github.com/openvinotoolkit/openvino/blob/b1ecd3ea8f10b057cd592e72e56b88efc401095d/src/bindings/python/src/openvino/runtime/opset1/ops.py#L1797-L1799

Reply:
Ok